elementor/elementor
# Fix lazyload mode still load hidden images from other gallery

## PR Checklist



## PR Type
What kind of change does this PR introduce?


## Summary

This PR can be summarized in the following changelog entry:

* Fix lazyload mode still load hidden images from other gallery. Issue: [#12065](https://github.com/elementor/elementor/issues/12065)

## Description
An explanation of what is done in this PR

* Fix lazyload mode still load hidden images from other gallery.
* Wrong hidden class name in lazyLoadImages function
* Issue: [#12065](https://github.com/elementor/elementor/issues/12065)

## Test instructions
This PR can be tested by following these steps:

* Build page with e-gallery widget multiple mode and lazyload checked.
* Load page preview, open network tab inspector from browser and see other hidden images from other gallery will not loaded until that gallery selected

## Quality assurance


Fixes #
